### PR TITLE
Aaru/advertising theme

### LINKED ISSUE #179: Advertising (new theme). Please review, test and commit
@sindhuPyxeda  - ticket for the new dataset. I have sent you both my encoded version and the original. Encoded version is attached here:
[advertising.zip](https://github.com/pyxeda/ML-Lessons/files/3388256/advertising.zip)

Doc: https://docs.google.com/document/d/1uygDdduQbN9oOmeyL0IaTwOAWT8hExow5YkmrSiJ9GQ/edit

### PR DESCRIPTION
**Pull request by AB3000**
*End-to-End testing is fine for both mock and replaced models along with manual testing (fixes #179). 

Some notes:
- UI box seems pretty long when I ran the python program. Goes top to bottom of screen height to capture all the fields (see below). I did not edit the UI code to maintain consistency. 
- In the .csv file, the predicted field (Spending on gadgets (encoded)) is not the last column, but for other datasets, as far as I saw, it was the last. 

Screen Shot 2019-07-16 at 10 58 42 AM

### CODE REVIEW

**Comment by davidanp** [comment]
quick fix for the mis alignment, give both the input field and the input field label a fixed width. You can make it smaller and force word wrap with css.

**Comment by AB3000** [comment]
Ah ok will do!

File: Advertising/projects/advertising_project_1.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    This function calculates the predictions on
+    multiple samples
+    Args:
+    values (dict): Dictionary containing url and feature values
+    fields (list): List of strings containing feature names
+    Returns:
+    predictions (list): Contains the list of predictions
+    """
+    values_age = values['Age']
+    values_split = values_age.split(',')
```
Comment (swamisundar) [inline Advertising/projects/advertising_project_1.py]: age_list

File: Advertising/projects/advertising_project_1.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+    arg_list = []
+
+    for a in range(0,len(values_split)):
```
Comment (swamisundar) [inline Advertising/projects/advertising_project_1.py]: a -> index

**Pull request merged**